src/test/java/org/synyx/urlaubsverwaltung/TestKeycloakContainer.java

```diff
@@ -0,0 +1,100 @@
+package org.synyx.urlaubsverwaltung;
+
+import dasniko.testcontainers.keycloak.KeycloakContainer;
+import jakarta.ws.rs.core.Response;
+import org.keycloak.OAuth2Constants;
+import org.keycloak.admin.client.Keycloak;
+import org.keycloak.admin.client.KeycloakBuilder;
+import org.keycloak.admin.client.resource.RealmResource;
+import org.keycloak.admin.client.resource.UserResource;
+import org.keycloak.admin.client.resource.UsersResource;
+import org.keycloak.representations.idm.CredentialRepresentation;
+import org.keycloak.representations.idm.UserRepresentation;
+import org.springframework.test.context.DynamicPropertyRegistry;
+
+import static org.keycloak.admin.client.CreatedResponseUtil.getCreatedId;
+
+public class TestKeycloakContainer extends KeycloakContainer {
+
+    private static final String VERSION = "22.0.4";
```

Comment:
How can we maintain this version here with dependabot?

Reply:
We cannot, we do this by hand, when we want to upgrade.